usr/share/rear/prep/USB/Linux-i386/350_check_usb_disk.sh

```diff
@@ -9,8 +9,8 @@ REAL_USB_DEVICE=$(readlink -f $USB_DEVICE)
 StopIfError "USB device '$USB_DEVICE' is not a block device"
 
 # Check if REAR-000 USB device is not accidentally mounted on other than $BUILD_DIR location.
-if grep -v $BUILD_DIR /proc/mounts | grep -q "^$REAL_USB_DEVICE"; then
-    Error "USB device '$REAL_USB_DEVICE' is already mounted."
+if res=( $(grep -v $BUILD_DIR /proc/mounts | grep "^$REAL_USB_DEVICE" ) ) ; then
+    Error "USB device '$REAL_USB_DEVICE' is already mounted on '${res[2]}'"
```

Comment:
most probably because outputs from `mount` and `cat /proc/mounts` differ a bit:

```
centos:(/root)(root)# tail -n1 /proc/mounts
tmpfs /run/user/0 tmpfs rw,nosuid,nodev,relatime,size=50012k,mode=700 0 0

centos:(/root)(root)# mount | tail -n1
tmpfs on /run/user/0 type tmpfs (rw,nosuid,nodev,relatime,size=50012k,mode=700)
```